netcheck/netcheck.go

```diff
@@ -460,17 +460,15 @@ func (c *Client) GetReport(ctx context.Context) (*Report, error) {
 
 	// Try HTTPS if UDP latency check failed for every stun server
 	// TODO: It does not show that UDP latency check failed
-	var (
-		rmu    sync.Mutex
-		wg     sync.WaitGroup
-		client = http.DefaultClient
-	)
-	client.Timeout = time.Second * 5
-
 	need := stuns4
-	if pc6 != nil && len(stuns6) > 0 {
+	if pc6 != nil {
 		need = append(need, stuns6...)
 	}
+	var (
+		wg  sync.WaitGroup // For HTTPS checks
+		rmu sync.Mutex     // For ret.DERPLatency
```

Comment:
that's what the existing `mu` mutex already guards. It's up above, right before the `ret` variable in a `var ( .... )` block.